config/locales/en.yml

```diff
@@ -420,6 +420,19 @@ en:
               title: Nothing to see here
               body: There are no invoices yet
         users:
+          index:
+            authorization_exists: yes
+            authorization_exists_not: not yet
+            delete_user_confirm: Are you sure you want to delete this user?
+            edit: Edit
+            invite_new_user: Invite a new user
+            empty_state:
+              title: No users yet
+              body: There are no users yet, and no invitations have been sent.
```

Comment:
Hm, are you sure that when there are no users in the list, there's always also no invitations?
I think if invitations have been sent out, but not accepted yet, the users list can be still empty (while on the invitations page there **will** be pending invitations).

Reply:
On that topic, I am not sure having zero users is even possible. At least there will be the current user, right?

Reply:
Oh, yes, that's actually true :sweat_smile: